backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/PostgresDocumentationUnitRepositoryImpl.java

```diff
@@ -367,6 +370,24 @@ public void saveProcedures(DocumentationUnit documentationUnit) {
     repository.save(documentationUnitDTO);
   }
 
+  @Override
+  @Transactional(transactionManager = "jpaTransactionManager")
+  public void saveLastPublicationDateTime(UUID uuid) {
+    if (uuid == null) {
+      return;
+    }
+
+    var documentationUnitDTOOptional = repository.findById(uuid);
+    if (documentationUnitDTOOptional.isEmpty()) {
+      return;
+    }
+    var documentationUnitDTO = documentationUnitDTOOptional.get();
+    ZoneId germanyZoneId = ZoneId.of("Europe/Berlin");
+    LocalDateTime nowInGermany = ZonedDateTime.now(germanyZoneId).toLocalDateTime();
+    documentationUnitDTO.setLastPublicationDateTime(nowInGermany);
+    repository.save(documentationUnitDTO);
```

Comment:
Open question: Should we use UTC in the backend and use timezones in the frontend?

Alternative idea: Use date/time completely without timezones. Consequences?

Reply:
Extracted to spike -> RISDEV-5528